src/PointwiseFunctions/GeneralRelativity/ExtrinsicCurvature.hpp

```diff
@@ -47,4 +48,59 @@ void extrinsic_curvature(
     const tnsr::ii<DataType, SpatialDim, Frame>& dt_spatial_metric,
     const tnsr::ijj<DataType, SpatialDim, Frame>& deriv_spatial_metric);
 /// @}
+
+/// @{
+/*!
+ * \ingroup GeneralRelativityGroup
+ * \brief  Computes the spatial covariant derivative of the extrinsic curvature.
+ *  \details The spatial covariant derivative is computed as
+ * \f[ D_k K_{ij} = \partial_k K_{ij} - {^{(3)}\Gamma^{l}_{ki}} K_{lj}
+ * - {^{(3)}\Gamma^{l}_{kj}}K_{il} \f]
+ * where \f$ {^{(3)}\Gamma^{k}_{ij}} \f$ is the spatial Christoffel symbol of
+ * the second kind.
+ */
+template <typename DataType, size_t SpatialDim, typename Frame>
+tnsr::ijj<DataType, SpatialDim, Frame>
+covariant_derivative_of_extrinsic_curvature(
+    const tnsr::ijj<DataType, SpatialDim, Frame>& d_ex_curv,
+    const tnsr::ii<DataType, SpatialDim, Frame>& ex_curv,
+    const tnsr::Ijj<DataType, SpatialDim, Frame>&
+        spatial_christoffel_second_kind);
+template <typename DataType, size_t SpatialDim, typename Frame>
+void covariant_derivative_of_extrinsic_curvature(
+    const gsl::not_null<tnsr::ijj<DataType, SpatialDim, Frame>*> grad_ex_curv,
+    const tnsr::ijj<DataType, SpatialDim, Frame>& d_ex_curv,
+    const tnsr::ii<DataType, SpatialDim, Frame>& ex_curv,
+    const tnsr::Ijj<DataType, SpatialDim, Frame>&
+        spatial_christoffel_second_kind);
+/// @}
+
+namespace Tags {
+template <size_t SpatialDim, typename Frame>
+struct CovariantDerivativeOfExtrinsicCurvatureCompute
```

Comment:
Can you add a dox string (just brief) so this can be discovered in the documentation? Maybe `\copydoc covariant_derivative_of_extrinsic_curvature` would work well here?